### ISSUE TITLE
autosleep may misleading stop unrelated apps in other spaces 

### ISSUE THREAD

**Issue by arnaudruffin** [Orange-OpenSource/autosleep#104]
Due to current CF-client-lib limitation, application stops are based on names and not guids. Seems to have possible cases where unrelated apps could be stopped in other spaces if they happen to have the same name than desired app to be put to sleep.

Wait for client-lib new version or contribute?

**Comment by antechrestos** [comment]
May bind wrong application too

``` java
client.bindService(application.getName(), cloudService.getName());
```

**Comment by arnaudruffin** [comment]
Shouldn't we open an issue on https://github.com/cloudfoundry/cf-java-client repo on these subjects?

**Comment by arnaudruffin** [comment]
Started patching v1.x branch to add stop/start/bind/unbind by GUID.
Shouldn't be too complicated once we manage to run the maven build commands.

see https://github.com/Orange-OpenSource/cf-java-client/tree/1.x

**Comment by arnaudruffin** [comment]
Apparently new CF-client-lib to be released won't be able to access recent logs through loggregator.
(see https://github.com/cloudfoundry/cf-java-client/issues/359#issuecomment-173403857).

We should agree on a workaround... Patch v1 with missing feature? Or try to solve new CF-client bug?

**Comment by antechrestos** [comment]
Fixed by #138
